internal/saucecloud/cloud.go

```diff
@@ -148,6 +150,13 @@ func (r *CloudRunner) collectResults(artifactCfg config.ArtifactDownload, result
 
 func (r *CloudRunner) runJob(opts job.StartOptions) (j job.Job, interrupted bool, err error) {
 	log.Info().Str("suite", opts.DisplayName).Str("region", r.Region.String()).Msg("Starting suite.")
+	if r.DryRun {
+		err := r.dryRunForMobile(opts)
```

Comment:
Why didn't you follow the existing convention of calling `dryRun` from `RunProject`? Isn't running `dryRunForMobile` for every **job** too granular?

Reply:
hmm. Let me check it again.

Reply:
Yeah this is mildly confusing, since dry run has nothing to do with mobile. All projects ought to support a dry run, regardless of the device or browser they use 🤔